src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -174,13 +174,13 @@ class RelationalDBInterfaceSqliteImp : public RelationalDBInterfaceSqlite
     bool
     transactionDbHasSpace(Config const& config) override;
 
-    int
+    std::uint64_t
     getKBUsedAll() override;
 
-    int
+    std::uint64_t
     getKBUsedLedger() override;
 
-    int
+    std::uint64_t
     getKBUsedTransaction() override;
 
```

Comment:
The use of these three functions in src/ripple/rpc/handlers/GetCounts.cpp needs updating to reflect the change from `int` to `std::uint64_t`.  Also in src/ripple/core/SociDB.h we're using `size_t` for these returns.  We should make these the same type alias.

Reply:
Thanks! I changed them all to `std::uint32_t`. Fixed in `975aebe35c [fold] Measure db memory usage with a uint32 var
`